docs/site/todo-tutorial-controller.md

```diff
@@ -74,7 +74,7 @@ export class TodoController {
     @repository(TodoRepository) protected todoRepo: TodoRepository,
   ) {}
 
-  @post('/todo')
+  @post('/todos')
```

Comment:
I think the singular name in the path was intentional here. `POST /todo` means "create a new TODO instance". `POST /todos` can be interpreted as "create many TODO instances".